Aztec/Classes/Private/Libxml2/Data/ElementNode.swift

```diff
@@ -903,6 +949,21 @@ extension Libxml2 {
             }
         }
 
+        func replace(range targetRange: NSRange, withNodeNamed nodeName: String, withAttributes attributes:[Attribute]) {
+            guard let textNode = lowestTextNodeWrapping(targetRange) else {
```

Comment:
The problem with this call is that it won't work for any range covering more than one node.  The ideal call here would be able to return more than one node for the specified range.

We can accept this for now, but would you mind creating a separate ticket to revisit this in the future?